Tests/SwiftAlgorithmsTests/TestUtilities.swift

```diff
@@ -62,7 +63,25 @@ func XCTAssertEqualSequences<S1: Sequence, S2: Sequence>(
   _ message: @autoclosure () -> String = "",
```

Comment:
We're not currently using this parameter since we're passing custom messages to the assertion functions in the implementation — let's change those to include the `message` parameter.